google_project-dns/variables.tf

```
@@ -0,0 +1,40 @@
+variable "app_name" {
+  description = "Application name or product name, e.g. autopush"
+  type        = string
+}
+
+variable "project_id" {
+  description = "GCP project_id where the zone will be provisioned."
+  type        = string
+}
+
+variable "parent_project_id" {
+  description = "GCP project_id that contains DNS zones used for delegation"
+  type        = string
+}
+
+variable "parent_managed_zone" {
+  description = "GCP DNS managed zone to add the record."
+  type        = string
+}
+
+variable "realm" {
+  description = "Realm is a grouping of environments being one of: global, nonprod, prod"
+  default     = ""
+  type        = string
+
+  validation {
+    condition     = contains(["global", "mgmt", "nonprod", "prod"], var.realm)
+    error_message = "Valid values for realm: global, mgmt, nonprod, prod."
+  }
+}
+
+variable "team_name" {
+  description = "Name of SRE team, which should correspond to the top-level folder name"
+  type        = string
+
+  validation {
+    condition     = contains(["cloudops", "dataops", "websre"], var.team_name)
+    error_message = "Valid values for team_name: cloudops, dataops, websre."
```

```suggestion
    error_message = "Valid values for team_name: servicessre, dataops, websre."
```

It's my understanding that we won't use cloudops, at least for now. We can also just have both cloudops and servicessre here as well.

---

yeah i'll add both to be safe, thanks!

---

@jasonthomas give it another review hwen you have a moment, thanks!